etcd/etcd-live-http-to-https-migration.md

```diff
@@ -1,73 +1,73 @@
+Repeat these steps on each etcd cluster member.
 
-### Generate TLS keypairs
+### Generate TLS Key Pairs
 
-Follow [generate self signed certificates][self-signed-ca] doc to generate self-signed certificates.
+Follow the guide to [generating self-signed certificates][self-signed-ca] to create the certificate/key pairs needed for each etcd cluster member.
 
-### Copy Keypairs
+### Copy Key Pairs to Nodes
 
-Let's assume we have three CoreOS hosts with three etcd cluster members: server1, server2, server3 and corresponding IP addresses: 172.16.0.101, 172.16.0.102, 172.16.0.103. We will use following keypairs filenames in our example:
+Assume we have three CoreOS machines, running three etcd cluster members: server1, server2, server3; with corresponding IP addresses: 172.16.0.101, 172.16.0.102, and 172.16.0.103. We will use the following key pair file names in our example:
 
 ```
 server1.pem
 server1-key.pem
 ```
 
-Create `/etc/etcd2` directory, copy there corresponding certificate and key and set file permissions:
+Create the `/etc/etcd2` directory, then copy the corresponding certificate and key there. Set permissions to secure the directory and key file:
 
 ```
-chown -R etcd:etcd /etc/etcd2/
-chmod 600 /etc/etcd2/*-key.pem
+$ chown -R etcd:etcd /etc/etcd2
+$ chmod 600 /etc/etcd2/*-key.pem
 ```
 
-Copy `ca.pem` CA certificate into `/etc/etcd2`.
+Copy the `ca.pem` CA certificate file into `/etc/etcd2` as well.
 
-Or copy your `ca.pem` CA certificate into `/etc/ssl/certs` directory and run `update-ca-certificates` script to update your certificates bundle. In this case CA will be available system wide and there will be not necessary to set CA path for each app.
+Alternatively, copy `ca.pem` into `/etc/ssl/certs` instead, and run the `update-ca-certificates` script to update the system certificates bundle. After doing so, the added CA will be available to any program running on the node, and it will not be necessary to set the CA path for each application.
 
-Repeat this step on rest of the member nodes.
+Repeat this step on the rest of the cluster members.
 
-### Using etcd Proxy
+### Using an etcd Proxy
 
-If you typically connect to an etcd cluster at a remote location, it's recommended to use this opportunity to configure an [etcd proxy][etcd proxy] that handles the remote connection logic and TLS termination, and reconfigure your apps to communicate through the proxy on localhost. In this case you have to generate client keypair (i.e. `client1.pem` and `client1-key.pem`) and follow the previous step.
+If you typically connect to a remote etcd cluster, this is a good time to configure an [etcd proxy][etcd proxy] that handles the remote connection and TLS termination, and to reconfigure your apps to communicate through the proxy on localhost. In this case, you must generate a client key pair (e.g. `client1.pem` and `client1-key.pem`) and repeat the *Copy Key Pairs* step above with the client key pair files.
```

Comment:
I'm not clear on what this means -- the proxy is going to accept insecure client connections and connect securely to the cluster?